src/main/java/hudson/plugins/sidebar_link/SidebarLinkPlugin.java

```diff
@@ -178,4 +183,9 @@ public boolean isAcceptedIconName(String iconName) {
         // accepts .gif extension as a compatibility with older version of the plugin
         return iconName.matches(("^[\\w-]+(\\.gif)?$"));
     }
+
+    private boolean isAcceptedSymbol(String iconName) {
+        return iconName.matches("(symbol-[-\\w]+(\\splugin-[-\\w]+)?|[-\\w]+)");
```

Comment:
That expression currently also matches things like `abc` and it allows spaces at the beginning which doesn't work.
How about `^symbol-[-\\w]+(\\s+plugin-[-\\w]+)?\\s*$`
This will only match strings that start with  `symbol-` and multiple spaces before plugin and spaces at the end are ok.

Reply:
Fixed

Reply:
Maybe some unit tests for this magic regexp?